### DISCUSSION TITLE
Are query projections supported?

### DISCUSSION THREAD

**Discussion by lnaie** [isar/isar#1055] [Help]
Hi,

I've seen this doc section: https://isar.dev/queries.html#property-queries about one property,
and I was wondering if multiple properties are supported.
Something like this -- where ```SlimTestModel``` has id, name and date as properties, a subset of the ```TestModel``` properties:
```
List<SlimTestModel> models = await isar.testModels.where()
  .nameProperty()
  .idProperty()
  .dateProperty()
  .findAll();
```

**Comment by JaonFanwt** [comment]
I have the same question, how to query part of the fields.
